lib/internal/process/next_tick.js

```diff
@@ -32,32 +32,58 @@ function setupNextTick() {
   const kHasScheduled = 0;
   const kHasPromiseRejections = 1;
 
-  const nextTickQueue = {
-    head: null,
-    tail: null,
+  // Queue size for each tick array. Must be a factor of two.
+  const kQueueSize = 2048;
+  const kQueueMask = kQueueSize - 1;
+
+  class FixedQueue {
+    constructor() {
+      this.btm = 0;
```

Comment:
I think the review comment that got deleted was

> Nit: Can you spell out `bottom` here?

Reply:
Sorry 😞